java-manta-client-unshaded/src/main/java/com/joyent/manta/config/ConfigContext.java

```diff
@@ -248,6 +254,7 @@ static String toString(final ConfigContext context) {
         sb.append(", verifyUploads=").append(context.verifyUploads());
         sb.append(", uploadBufferSize=").append(context.getUploadBufferSize());
         sb.append(", skipDirectoryDepth=").append(context.getSkipDirectoryDepth());
+        sb.append(", pruneEmptyParentDepth=").append(context.getPruneEmptyParentDepth());
         sb.append(", metricReporterMode=").append(context.getMetricReporterMode());
```

Comment:
I noticed the downloadContinuations setting wasn't being output here so I edited it in master. That _might_ cause a merge conflict (which I can resolve through the PR).